styles.css

```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  display: flex;
+  height: 15rem;
+  padding: 5% 3% 3% 8%;
+  border: none;
+  justify-content: space-between;
+}
+
+h1 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-style: italic;
+  font-size: 1.5rem;
+  color: rgb(0, 179, 60);
+}
+
+h2 {
+  font-family: 'Ibarra Real Nova', serif;
+}
+
+h3 {
+  font-family: 'Ibarra Real Nova', serif;
+  font-size: 1.5rem;
+}
+
+h4 {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.container {
+  width: 80%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.intro {
+  padding-right: 2rem;
+  font-size: 20px;
+}
+
+.card {
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.hero-image {
+  background-image: url(images/header-image.jpg);
+}
+
+nav {
+  width: 100%;
+  background-color: rgb(240, 240, 240);
+  padding-left: 18rem;
+  margin-bottom: 2rem;
+  float: left;
+}
+
+.dog-card {
+  width: 20rem;
+  border: solid;
+  border-color: rgb(240, 240, 240);
+  margin-right: 1rem;
+  margin-bottom: 1rem;
+}
+
+.dog {
+  width: 100%;
+}
+
+.dog-button {
+  width: 100%;
+  background-color: rgb(0, 179, 60);
+  border: none;
+  align-items: center;
+  color: white;
+  cursor: pointer;
+}
+section {
+  padding-left: 20rem;
+  padding-right: 2rem;
+}
+
+footer {
+  background-color: rgb(64, 64, 64);
+  height: 20%;
+  width: 100%;
+  padding-top: 3rem;
+  padding-bottom: 3rem;
+  margin-top: 1.5rem;
+  align-items: center;
+  color: white;
+  position: relative;
+  text-align: center;
+  z-index: -1;
+}
+
+button, input, textarea {
+  display: block;
+  margin-bottom: 1rem;
+  padding: .5rem;
+  font-family: inherit;
+  font-size: 100%;
+  width: 80%;
+  outline: inherit;
+}
+
+select {
+  display: block;
+  width: 20%;
+  border-radius: 0;
+}
+
+fieldset  {
+  border: none;
+  width: 20%;
+}
+
+fieldset  input {
+  display: inline;
+  width: 15%;
+}
+
+.blog {
+  margin-bottom: 3rem;
+  margin-right: 1rem;
+  float: left;
+}
+
+.clear {
+  clear: both;
+}
+
+.margin-right{
+  margin-right: 1rem;
+}
+
+.margin-top {
+  margin-top: 1rem;
+}
+
+.margin-bottom {
+  margin-bottom: 6rem;
+}
+
+a:link {
+  text-decoration: none;
+}
+
+a:hover {
+  color: rgb(0, 179, 60);
+}
+
+a:visited {
+  text-decoration: none;
+}
+
+a::before {
+  content: "/  "
+}
+
+a.done::before {
```

Comment:
instead of adding a class to not show the `/` maybe use nth-child instead. There also is a method using `not`, but for now I'd work on the first/last/nth-child method